standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -248,199 +182,141 @@
  * the metstore call stack should have logic not to retry.  There are {@link RetrySemantics}
  * annotations to document the behavior.
  */
+@SuppressWarnings("SqlSourceToSinkFlow")
 @InterfaceAudience.Private
 @InterfaceStability.Evolving
-abstract class TxnHandler implements TxnStore, TxnStore.MutexAPI {
+public abstract class TxnHandler implements TxnStore, TxnStore.MutexAPI {
+
   
-  private static final String TXN_TMP_STATE = "_";
-  private static final String DEFAULT_POOL_NAME = "default";
+  public static class TxnHandlingFeatures {    
```

Comment:
please rename to ConfVars. 
````
ConfVars.init(table->checkIfTableIsUsable(table), conf)

public static final ConfVars {
  private boolean useMinHistoryLevel;
  private boolean useMinHistoryWriteId;
  
  public init(Predicate<String> tableCheck, Conf conf){
    useMinHistoryLevel = tableCheck.test("MIN_HISTORY_WRITE_ID", 
  MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.TXN_USE_MIN_HISTORY_WRITE_ID))
  }
}
````

ConfVars.get().useMinHistoryWriteId
ConfVars.get().useMinHistoryLevel

Reply:
done, except I did not remove the getters as they are used outside of `TxnHandler` too.